src/qt/pivx/pwidget.cpp

```diff
@@ -0,0 +1,73 @@
+#include "qt/pivx/pwidget.h"
+#include "qt/pivx/qtutils.h"
+#include "qt/pivx/moc_pwidget.cpp"
```

Comment:
I haven't fully proven out my environment; but I got hung up on not finding this file.

Reply:
Need more context.

Are you compiling it deterministically or using your system's local dependencies? Linux, Windows OSX?

Please perform a `git clean -dfx` and get rid of everything there first, then compile it again.

Reply:
I was building on Ubuntu 18.04.  I started a clean fresh build before I left for work, so I'll look again when I get home.

Reply:
I thought the problem might be because I didn't have qtsvg or qtcharts in my depends.   That problem is gone, but this one still exists:
Ubuntu 18.04
```
cd depends
make HOST=x86_64-pc-linux-gnu -j4
cd ..
./autogen.sh
./configure --enable-tests=no --prefix `pwd`/depends/$CROSS
make install
```
```
qt/pivx/pwidget.cpp:3:10: fatal error: qt/pivx/moc_pwidget.cpp: No such file or directory
 #include "qt/pivx/moc_pwidget.cpp"
          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
Makefile:8812: recipe for target 'qt/pivx/qt_libbitcoinqt_a-pwidget.o' failed
```
Is moc_pwidget.cpp supposed to exist in src/qt/pivx/.  If not, where is it supposed to be?

Reply:
I have been reading up on moc, and seeing the function in the travis build logs.  I'll do some more checking on my end, since it looks like it's something with my environ.

Reply:
ok, have a convo today about this with warrows, it seems that some OSs are not creating / locating properly the moc files. Will create a debian vm and see why it's happening after finish with the corrections.